src/app/pages/applications/dialogs/chart-events/chart-events-dialog.component.html

```diff
@@ -0,0 +1,78 @@
+<div mat-dialog-content fxLayout="column wrap" fxLayoutAlign="center start" >
+    <div class="logo-container" fxLayout="row" fxLayoutAlign="center center">
+        <div class="logo" style="background-image: url('{{ catalogApp.icon }}');"></div>
+        <div>
+            <strong class="chart-name">{{ catalogApp.name }}</strong><span class="version"> {{ catalogApp.version}}</span>
+        </div>
+    </div>
+    <div class="ports">
+        <span>Used Ports:</span><span class="value" textLimiter threshold="70">{{ catalogApp.used_ports }}</span>
```

Comment:
The most of strings in this file can't be translated to other langs

Reply:
> The most of strings in this file can't be translated to other langs

Fixed